app/repository/CovidReportRepository.ts

```diff
@@ -36,6 +39,16 @@ export class CovidReportRepository {
     return jsonDump ? this.parseJsonDumpToCovidReport(jsonDump) : undefined;
   }
 
+  async getAllCovidReports(): Promise<CovidReport[]> {
+    const results = await (this.db?.getAll(SELECT_ALL_COVID_REPORTS, [
+    ]));
+    const ret = []
```

Comment:
Kan byttes ut med `return results.map(result =>  this.parseJsonDumpToCovidReport(result.json_dump))`

Reply:
:+1: